.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+Based on the nature of your change, please use the relevant template below and modify to suit your needs.
```

Comment:
Yeah, this is the one thing I wasn't sure about. We could streamline this to a simple one-line instruction?